### PR TITLE
AlexBear.py: Change AlexBear docstring

### PR DESCRIPTION
**Pull request by cosmin004**
The docstring mentions that you should
use only Englis language.
It also includes a link to Alex for
instructions and tests.

Fixes https://github.com/coala/coala-bears/issues/1138

### CODE REVIEW

**Comment by sims1253** [comment]
Maybe reword the "In the *.md file it should be English language only. " sentence to something like "Be aware that alex and this bear only work on english text."

**Comment by sims1253** [comment]
Also the Fixes should be a Closes as this is not a bugfix

**Comment by sims1253** [comment]
looks good. Though "something like" does not mean "copy what I suggested" ;)

**Comment by sims1253** [comment]
ack d53fdcd

**Comment by sims1253** [comment]
unack d53fdcd

you have trailing whitespaces

File: bears/natural_language/AlexBear.py [hunk 1]
```diff
@@ -14,6 +14,10 @@ class AlexBear:
     """
     Checks the markdown file with Alex - Catch insensitive, inconsiderate
     writing.
+    
+    Be aware that Alex and this bear only work on english text.
```
Comment (jayvdb) [inline bears/natural_language/AlexBear.py]: English is a proper noun and should be capitalised

**Comment by jayvdb** [comment]
The commit msg is incorrect. Using English is not recommended.
This bear only *supports* English.

**Comment by jayvdb** [comment]
Bare links are not ideal. Use RST syntax in docstrings

**Comment by sims1253** [comment]
ack 6b69b22

**Comment by sims1253** [comment]
@rultor merge

**Comment by sims1253** [comment]
@sils help

**Comment by cosmin004** [comment]
@jayvdb can you please accept the changes if they were done?

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1149/commits/cd89a1d6cc63f0d02aaf954805ffaf350993d918

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
Hey, this is your first contribution, right? Congrats and welcome to the coalaians! :)

You can help us a lot by filling out <http://coala.io/newform>!

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

See also https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer btw.

We would really appreciate it if you could help us with

- some code reviews for other newcomers
- fixing at least a difficulty/low issue

We're all volunteers and we're currently struggling to keep this up. Helping all those newcomers is a lot of work and we do need your help so we can continue this for others!
